### PR TITLE
Placeholder pages and navigation SetUp

### PR DESCRIPTION
**Pull request by JoshuaGomersall**

### CODE REVIEW

**Comment by tvaidotas** [comment]
- Split up the HomePage controller into respective page controllers
- Add parentheses for the methods
- Extract the movie list into a helper object
- format the code with CTRL + ALT + L
- Remove all the boiler place HTML and use main template

File: app/views/allFilmsGallery.scala.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+@(implicit test :List[String] = List("Jaws 3D","Titan Fall 2","Film 3","Film 4" ,"Film 5","Lego Star Wars"))
+
+@main("QA Cinema") {
+    <h1>All Film DB</h1>
+
+    <div class="flex">
+    @for(i <- test.length-5 to 1 by -1){
+        @println(i)
+        @if(i % 3==0){
+        </div>
```
Comment (dsmith2708) [inline app/views/allFilmsGallery.scala.html:10]: add logic here to fix top row not being full. potentially if this is the first time the for loop is running get the remainder of the total/3 and only add this number to the bottom row so the top row is full

File: app/views/allFilmsGallery.scala.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    <div class="flex">
+    }
+        <div class="filmpanel">
+            <h3>@test(i-1) </h3>
```
Comment (dsmith2708) [inline app/views/allFilmsGallery.scala.html]: this shouldn't work?

**Review by dsmith2708 — COMMENTED** [review]
See comments left about code that shouldn't work and refactor all films to fill top row

File: app/controllers/AboutUs.scala [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package controllers
+
+import play.api._
+import play.api.mvc._
+
+class AboutUs extends Controller {
+
+
+  val allFilmsList :List[String] = List[String]("1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18")
```
Comment (tvaidotas) [inline app/controllers/AboutUs.scala]: get rid of this

File: app/controllers/HomePage.scala [hunk 4]
```diff
@@ -9,4 +9,8 @@ class HomePage extends Controller {
     Ok(views.html.index("Your new application is ready."))
   }
 
+  def errorPage = Action {
```
Comment (tvaidotas) [inline app/controllers/HomePage.scala]: lift it out

File: public/javascripts/demo_create_mongo_db.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+var MongoClient = require('mongodb').MongoClient;
```
Comment (tvaidotas) [inline public/javascripts/demo_create_mongo_db.js]: get rid of this

File: public/javascripts/demo_mongodb_createcollection.js [hunk 6]
```diff
@@ -0,0 +1,13 @@
+﻿var MongoClient = require('mongodb').MongoClient;
```
Comment (tvaidotas) [inline public/javascripts/demo_mongodb_createcollection.js]: get rid of this

File: public/javascripts/demo_mongodb_find.js.js [hunk 7]
```diff
@@ -0,0 +1,12 @@
+var MongoClient = require('mongodb').MongoClient;
```
Comment (tvaidotas) [inline public/javascripts/demo_mongodb_find.js.js]: get rid of this

File: public/javascripts/demo_mongodb_insert.js [hunk 8]
```diff
@@ -0,0 +1,15 @@
+var MongoClient = require('mongodb').MongoClient;
```
Comment (tvaidotas) [inline public/javascripts/demo_mongodb_insert.js]: get rid of this

**Review by tvaidotas — COMMENTED** [review]
fix comments pls

**Pull request merged**